gyft.py

```diff
@@ -126,31 +126,43 @@ def merge_slots(in_dict):
     for time in timetable_dict[day]:
         timetable_dict[day][time].append(" ")
         course_code = timetable_dict[day][time][0]
-        course_dept = course_code[:2]
+        course_dept = course_code[:2] 
         if course_dept not in courses.keys():
             courses[course_dept] = {}
         if course_code not in courses[course_dept].keys():
             courses[course_dept][course_code] = ''
 
 # scraping course names deptwise
 for dept in courses.keys():
-    DEPT_URL = COURSES_URL.format(dept)
-    r = s.get(DEPT_URL, headers=headers)
-    soup = bs(r.text, 'html.parser')
-    parentTable = soup.find('table', {'id': 'disptab'})
+    try: 
+        if dept == 'EP': # special case -> EP courses are under RJ department, not EP department
+            DEPT_URL = COURSES_URL.format('RJ')
+        else:
+            DEPT_URL = COURSES_URL.format(dept)
```

Comment:
```suggestion
        if dept == 'EP': # special case -> EP courses are under RJ department, not EP department
            DEPT_URL = COURSES_URL.format('RJ')
        else:
            DEPT_URL = COURSES_URL.format(dept)
```
Use a dict mapping the Incorrect code to the correct code. If required create a separate file for this purpose.